web/app.js

```diff
@@ -405,7 +405,7 @@ const PDFViewerApplication = {
       eventBus = new EventBus();
     }
     if (this.mlManager) {
-      this.mlManager.eventBus = eventBus;
+      this.mlManager.setEventBus(eventBus, this._globalAbortController.signal);
     }
```

Comment:
This can be simplified, via optional chaining, now that we're using a method to set the eventBus.
```javascript
this.mlManager?.setEventBus(eventBus, this._globalAbortController.signal);
```